src/test/java/org/fundacionjala/coding/abel/CodeCesarTest.java

```diff
@@ -0,0 +1,96 @@
+package org.fundacionjala.coding.abel;
+
+        import org.junit.Test;
+
+        import static org.junit.Assert.assertEquals;
+
+
+/**
+ *
+ */
+public class CodeCesarTest {
+
+    private static final int NUM = 5;
+    private static final int NUM2 = 31;
+
+
+    private static final String CAD1 = "ZORRO O";
+    private static final String CAD2 = "ETWWT T";
+
+    private static final String CAD5 = "ZORRO O=";
+    private static final String CAD6 = "ETWWT T=";
+    private static final String CAD3 = "ZORRO";
+    private static final String CAD4 = "ETWWT";
+    private static final String CAD7 = "909 O";
+    private static final String CAD8 = "909 T";
+
+    /**
+     *
+     */
+    @Test
+    public void test1() {
+        CodeCesar k = new CodeCesar();
```

Comment:
@AT-08/at-08 
You are repeating this line of code in all your unit tests.
Please move it to an @before method.